app/src/main/java/com/android/calendar/persistence/tasks/AstridCloneTasksContract.java

```diff
@@ -0,0 +1,34 @@
+package com.android.calendar.persistence.tasks;
+
+import android.net.Uri;
+import android.provider.BaseColumns;
+
+public class AstridCloneTasksContract  implements BaseColumns {
```

Comment:
looks rather non functional, either squash it with the implementation or drop it from the PR.